app/src/users/admin.py

```diff
@@ -116,6 +116,44 @@ def queryset(self, request, queryset):
         return queryset
 
 
+class UserAccommodationListFilter(admin.SimpleListFilter):
+    title = 'user accommodation'
+    parameter_name = 'user_accommodation'
+
+    def lookups(self, request, model_admin):
+        return [
+            ('no', 'No accommodation'),
+            ('not_continuous', "Not continuous accommodation"),
```

Comment:
Please decide on quotes used ;)

Reply:
I have used PaymentAcceptedListFilter as inspiration and I didn't notice it was mixed there too :upside_down_face: 
 https://github.com/ksiuwr/zosia16-site/blob/1c425dc7017ec3df320d64b5cb56a9aaa56d07e5/app/src/users/admin.py#L105-L108